### ISSUE TITLE
allow to use `buildBlockExplorer` without any param

### ISSUE THREAD

**Issue by LuizAsFight** [FuelLabs/fuels-ts#752]
### Motivation

Currently when using `buildBlockExplorer`, the user needs to inform some params, or the function will throw an error [here](https://github.com/FuelLabs/fuels-ts/blob/cef77013e14461b07a525213046859ba1013dd29/packages/providers/src/util.ts#L175)

then in front-end I'm needing to workaround, passing a empty space in path
```
    buildBlockExplorerUrl({
      providerUrl: currentNetworkUrl,
      path: ' ',
    }),
``` 

In my case I'm passing a custom `providerUrl` , but even only `buildBlockExplorerUrl()` should just work only the correct block-explorer URL

**Comment by Dhaiwat10** [comment]
@LuizAsFight can you share the code snippet from your frontend if possible? The code that's giving you an error

**Comment by LuizAsFight** [comment]
@Dhaiwat10 
I guess it's not a bug, it was made to work on this way

this is the code:
```
buildBlockExplorerUrl({
      providerUrl: currentNetworkUrl,
});
```

**Comment by Dhaiwat10** [comment]
Hey @LuizAsFight yeah that will throw. What is the result you are expecting from that code snippet? Just `https://fuellabs.github.io/block-explorer-v2?providerUrl=<custom_url>`? 

Should I add a condition in the code that ensures that the function does not throw if a `providerUrl` is passed even if neither `path` or any of the helper params are passed?

**Comment by LuizAsFight** [comment]
yes, no needs to be strict on this input params.. I opt for flexibility, and let the user validate by himself if his params are right for his specifics requirements

so IMO this is necessary:
```
buildBlockExplorerUrl({
      providerUrl: currentNetworkUrl,
});
// result should be: `https://fuellabs.github.io/block-explorer-v2?providerUrl=<custom_url>`
```


additionally, even this should just work:
```
buildBlockExplorerUrl();
// result should be: `https://fuellabs.github.io/block-explorer-v2`
```
